src/pages/product-list-page/product-list-filter/price-filter/price-filter.js

```diff
@@ -17,15 +18,16 @@ const PriceFilter = ({ priceRange }) => {
   const searchParams = new URLSearchParams(search);
   const styles = useStyles();
   const history = useHistory();
+  const priceFilterValue = searchParams.get(priceFilter);
 
-  const [prices, setPrices] = useState(
-    searchParams.get(priceFilter)
-      ? searchParams
-        .get(priceFilter)
-        .split(',')
-        .map((price) => +price)
-      : ['', '']
-  );
+  const getDefaultPrices = () => priceFilterValue
```

Comment:
Use react-router for params.

Reply:
We already use react-router `useLocation() ` hook and after parsing it via `new URLSearchParams()` object. It's the way how it work in react-router v5 which we currently using on our project. In react-router v6 new hook `useSearchParams()` was added but currently such as we using a lower version of library we can't use it.